src/host/main.cpp

```diff
@@ -62,19 +62,6 @@ void print_version(size_t)
   exit(0);
 }
 
-std::string read_required_environment_variable(
-  const std::string& envvar, const std::string& name)
-{
-  auto ev = std::getenv(envvar.c_str());
-  if (ev == nullptr)
-  {
-    LOG_FATAL_FMT(
-      "Environment variable \"{}\" for {} is not set", envvar, name);
-  }
-  LOG_INFO_FMT("Reading {} from environment {}", name, envvar);
-  return ev;
-}
```

Comment:
Unused.

Reply:
I think that was once used for UVM endorsements, but that has all moved to configuration now.